src/views/tutorial-view/index.tsx

```diff
@@ -101,6 +102,9 @@ export default function TutorialView({
     video,
     collectionCtx,
   } = tutorial
+  const featuredInWithoutCurrent = collectionCtx.featuredIn?.filter(
```

Comment:
Moving this filtering up to the frontend so that the featured collections can be consistent (including the current). The featured collections are what we generate the canonical url from if its default collection is not in beta.